### ISSUE TITLE
Hello sir, i got a bug just like in the picture below, may i ask you why? Thank you so much

### ISSUE THREAD

**Issue by whistledev0910** [dan5py/turborepo-shadcn-ui#11]
![Screenshot 2023-09-13 at 18 35 37](https://github.com/dan5py/turborepo-shadcn-ui/assets/88497295/ff7fff20-935e-4b9e-89a9-e38fe82b03d5)

The error said
error ../../packages/ui/src/components/ui/accordion.tsx:7:0
shell:dev: Module not found: Can't resolve '@ui/lib/utils'
shell:dev:    5 | import { ChevronDown } from 'lucide-react'
shell:dev:    6 | 
shell:dev: >  7 | import { cn } from '@ui/lib/utils'
shell:dev:    8 | 
shell:dev:    9 | const Accordion = AccordionPrimitive.Root
shell:dev:   10 | 
shell:dev: 
shell:dev: https://nextjs.org/docs/messages/module-not-found

**Comment by whistledev0910** [comment]
I followed your instructions and only seemed to change the tsconfig.json file because my folder structure was surrounded by the src folder.

My tsconfig.json

```
{
  "extends": "tsconfig/react-library.json",
  "include": ["."],
  "exclude": ["dist", "build", "node_modules"],
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@ui/*": ["./src/*"],
      "@ui/components/ui/*": ["./src/components/ui/*"]
    }
  }
}
```

**Comment by dan5py** [comment]
This should be fixed with the latest relase `1.1.0`
